encord/client.py

```diff
@@ -392,6 +394,23 @@ def get_project(self):
         """
         return self._querier.basic_getter(Project)
 
+    def get_filtered_label_rows(
```

Comment:
I don't think we need the 'filtered' in the name. If you call this with no parameters in will give you all the label rows, right?

Reply:
We are leaving get_label_rows for a different function. So this was renamed on purpose. 
Now that the filters are optional I'll check with @ramnik-cord  if we can rename this function to get_label_rows.

Reply:
I'm slightly confused. What would that "different function" called `get_label_rows` do that is different from what this does? 

I don't see a need for 'just list, don't expant' method, because label_rows are included with the project info, AFAIR.